ui/src/app/pages/login/account-creation/account-creation-tos.tsx

```diff
@@ -82,7 +80,9 @@ export const AccountCreationTos = withWindowSize()(
       }
       this.hasCreatedIntersectionObserver = true;
       const intersectionCallback: IntersectionObserverCallback = (
-        entries: IntersectionObserverEntry[], unusedObserver: IntersectionObserver) => {
+        entries: IntersectionObserverEntry[],
```

Comment:
the rest of the stuff in `handleLastPageRender()` looks like stuff that might be able to go into the shared component.

I don't fully understand how it works but I think, ultimately, components that use the PdfViewer component just want to know when the user has read the entire PDF and not concern themselves with stuff like `IntersectionObserverCallback`.